src/lobby.js

```diff
@@ -46,20 +46,12 @@ class Lobby extends React.Component {
   }
 
   fetchRooms() {
-    return Promise.all(
-      Config.public_rooms.map(room_id => {
-        return this.handle
-          .sendMessage({
-            kind: "listusers",
-            room_id
-          })
-          .then(signal => ({
-            id: room_id,
-            limit: 12,
-            users: signal.plugindata.data.response.user_ids
-          }));
-      })
-    );
+    return this.handle
+      .sendMessage({ kind: "listusers" })
+      .then(signal => {
+        const usersByRoom = signal.plugindata.data.response.users;
+        return Config.public_rooms.map(id => ({id, limit: 12, users: usersByRoom[id] || []}));
```

Comment:
We could actually list all the occupied rooms with the new Janus API, but I assumed we want to keep the lobby list as it is (with rooms 1-5 pre-defined). This effectively makes all other rooms pseudo-private, or at least unlisted from the UI.

Reply:
Yeah, I think that's good enough for now.

Reply:
Yeah, the pseudo privacy is why I did it this way. Doesn't seem worth doing this in Janus as we probably want to handle all room listing and such in reticulum eventually, so this is good enough for now.